docs/architecture/adr-063-core-module-api.md

```diff
@@ -166,6 +166,29 @@ the block or app hash is left to the runtime to specify).
 Events emitted by `EmitKVEvent` and `EmitProtoEventNonConsensus` are not considered to be part of consensus and cannot be observed
 by other modules. If there is a client-side need to add events in patch releases, these methods can be used.
 
+#### Logger
+
+A logger (`cosmossdk.io/log`) must be supplied using `depinject`, and will
+be made available for modules to use via `depinject.In`.
+Modules using it should follow the current pattern in the SDK by adding the module name before using it.
+
+```go
+type ModuleInputs struct {
+  depinject.In
+
+  Logger log.Logger
+}
+
+func ProvideModule(in ModuleInputs) ModuleOutputs {
+  // add the module name to the logger
+  logger := in.Logger.With("module", "x/"+types.ModuleName)
```

Comment:
This makes sense to me the way it is explained, but our modules will do it in the keepers directly, right?
If so, then maybe we should add a small remark here, as this description won't correspond to what we do.

Reply:
Yes, I'll update it. We do it in the NewKeeper function instead, so it applies to every user instead of just for depinject users